plugins/source/aws/resources/services/efs/filesystems.go

```diff
@@ -82,3 +87,26 @@ func ResolveEfsFilesystemBackupPolicyStatus(ctx context.Context, meta schema.Cli
 
 	return resource.Set(c.Name, response.BackupPolicy.Status)
 }
+
+func ResolveEfsFilesystemPolicy(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
```

Comment:
```suggestion
func resolveEfsFilesystemPolicy(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
```

This should be a private function and not available outside of the package

Reply:
Done, I also change another function to the lower case as well because I see that function also a private funciton.